### PR TITLE
Rivendare new kit done

### PR DESCRIPTION
**Pull request by SebastianTanoni**

### CODE REVIEW

File: mapdata/WarcraftLegacies/AbilityData/Skin/1395667265.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Summons Ramstein the Gorger, a mighty abomination.|nHas Frenzy, Cleaving Attack and Bash.|nLast 60 seconds."
+      "Value": "Summons Ramstein the Gorger, a mighty abomination.|nHas Frenzy, Cleaving Attack, Aura of Dread and Bash.|nLast 60 seconds."
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1395667265.json:70]:
```suggestion
      "Value": "Summons Ramstein the Gorger, a mighty abomination.|nHas Frenzy, Cleaving Attack, Aura of Dread, and Bash.|nLasts 60 seconds."
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    {
+      "Id": 1952805473,
+      "Type": 3,
+      "Value": "Learn Army of the Dead - [|cffffcc00Level %d|r]"
+    },
+    {
+      "Level": 1,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Army of the Dead- [|cffffcc00Level 1|r]"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json:43]:
```suggestion
      "Value": "Army of the Dead - [|cffffcc00Level 1|r]"
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    },
+    {
+      "Level": 2,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Army of the Dead- [|cffffcc00Level 2|r]"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json:49]:
```suggestion
      "Value": "Army of the Dead - [|cffffcc00Level 2|r]"
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    },
+    {
+      "Level": 3,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Army of the Dead- [|cffffcc00Level 3|r]"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json:55]:
```suggestion
      "Value": "Army of the Dead - [|cffffcc00Level 3|r]"
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    },
+    {
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Summon 1 Dark Minion every time the hero casts a spell. Lasts 45 seconds."
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json:61]:
```suggestion
      "Value": "Causes the hero to summon 1 Dark Minion every time it casts a spell. The minion lasts 45 seconds."
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    },
+    {
+      "Level": 2,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Summon 2 Dark Minion every time the hero casts a spell. Lasts 45 seconds."
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json:67]:
```suggestion
      "Value": "Causes the hero to summon 2 Dark Minions every time it casts a spell. The minions lasts 45 seconds."
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+    },
+    {
+      "Level": 3,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Summon 3 Dark Minion every time the hero casts a spell. Lasts 45 seconds."
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json:73]:
```suggestion
      "Value": "Causes the hero to summon 3 Dark Minions every time it casts a spell. The minions lasts 45 seconds."
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+    {
+      "Id": 1835101793,
+      "Type": 3,
+      "Value": "Army of the Dead"
+    },
+    {
+      "Level": 4,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Summon 4 Dark Minion every time the hero casts a spell. Lasts 45 seconds."
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json:84]:
```suggestion
      "Value": "Causes the hero to summon 4 Dark Minions every time it casts a spell. The minions last 45 seconds."
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+  "Modifications": [
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(scourge)"
+    },
+    {
+      "Id": 1953854049,
+      "Type": 3,
+      "Value": "Rivendare raise skeletal minions every time he casts a spell. They last 45 seconds.|n|n|cffffcc00Level 1|r - 1 minion|n|cffffcc00Level 2|r - 2 minion|n|cffffcc00Level 3|r - 3 minion|n|cffffcc00Level 4|r - 4 minion"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json:16]:
```suggestion
      "Value": "Causes the hero to summon Dark Minions every time it casts a spell. The minions last 45 seconds.|n|n|cffffcc00Level 1|r - Summons 1 minion.|n|cffffcc00Level 2|r - Summons 2 minions.|n|cffffcc00Level 3|r - Summons 3 minions.|n|cffffcc00Level 4|r - Summons 4 minions."
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+    },
+    {
+      "Level": 4,
+      "Id": 829453409,
+      "Type": 3,
+      "Value": "Army of the Dead- [|cffffcc00Level 4|r]"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/842355795.json:22]:
```suggestion
      "Value": "Army of the Dead - [|cffffcc00Level 4|r]"
```

**Comment by YakaryBovine** [comment]
Dark Grip has no debuff.
image

File: mapdata/WarcraftLegacies/AbilityData/Skin/1496920129.json [hunk 11]
```diff
@@ -57,19 +57,19 @@
       "Level": 1,
       "Id": 829453409,
       "Type": 3,
-      "Value": "Horn of Stormwind - [|cffffcc00Level 1|r]"
+      "Value": "Death\u0027 Advance - [|cffffcc00Level 1|r]"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1496920129.json:60]:
```suggestion
      "Value": "Death\u0027s Advance - [|cffffcc00Level 1|r]"
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/1496920129.json [hunk 12]
```diff
@@ -57,19 +57,19 @@
...
     },
     {
       "Level": 2,
       "Id": 829453409,
       "Type": 3,
-      "Value": "Horn of Stormwind - [|cffffcc00Level 2|r]"
+      "Value": "Death\u0027 Advance - [|cffffcc00Level 2|r]"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1496920129.json:66]:
```suggestion
      "Value": "Death\u0027s Advance - [|cffffcc00Level 2|r]"
```

File: mapdata/WarcraftLegacies/AbilityData/Skin/1496920129.json [hunk 13]
```diff
@@ -57,19 +57,19 @@
...
     },
     {
       "Level": 3,
       "Id": 829453409,
       "Type": 3,
-      "Value": "Horn of Stormwind - [|cffffcc00Level 3|r]"
+      "Value": "Death\u0027 Advance - [|cffffcc00Level 3|r]"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1496920129.json:72]:
```suggestion
      "Value": "Death\u0027s Advance - [|cffffcc00Level 3|r]"
```

**Comment by YakaryBovine** [comment]
(Suggestion, this is not part of the review process)
image
Spell Immunity here feels a bit cheap. Feels like there's been a lot of temptation lately to give Spell Immunity to summons to deal with their incredible weakness to Dispel Magic... But why have Dispel Magic working on summons at all at that point? Think there's a deeper problem to address here instead of slapping Spell Immunity on random non-Ultimate summons.

**Comment by YakaryBovine** [comment]
Right now, Army of the Dead always summons the Dark Minions 200 units to the right of Rivendare, which feels a bit weird. 
Two options to fix this:
* Reduce the area of effect of the Summon Water Elemental dummy from 200 to 0, so it spawns right on top of Rivendare
* We figure out how to make the dummy used by Army of the Dead face the same way as Rivendare
image

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
just a few things to fix!

File: mapdata/WarcraftLegacies/AbilityData/Skin/1496920129.json (hunk 11 above)
Comment (SebastianTanoni) [reply mapdata/WarcraftLegacies/AbilityData/Skin/1496920129.json:60]: Why the /u0027s?

**Comment by SebastianTanoni** [comment]
> Right now, Army of the Dead always summons the Dark Minions 200 units to the right of Rivendare, which feels a bit weird. Two options to fix this:
> 
> * Reduce the area of effect of the Summon Water Elemental dummy from 200 to 0, so it spawns right on top of Rivendare
> * We figure out how to make the dummy used by Army of the Dead face the same way as Rivendare
>   image

made the range 0 here

**Comment by SebastianTanoni** [comment]
> (Suggestion, this is not part of the review process) image Spell Immunity here feels a bit cheap. Feels like there's been a lot of temptation lately to give Spell Immunity to summons to deal with their incredible weakness to Dispel Magic... But why have Dispel Magic working on summons at all at that point? Think there's a deeper problem to address here instead of slapping Spell Immunity on random non-Ultimate summons.

gave the unit magic resistance instead

**Comment by YakaryBovine** [comment]
> > Right now, Army of the Dead always summons the Dark Minions 200 units to the right of Rivendare, which feels a bit weird. Two options to fix this:
> > 
> > * Reduce the area of effect of the Summon Water Elemental dummy from 200 to 0, so it spawns right on top of Rivendare
> > * We figure out how to make the dummy used by Army of the Dead face the same way as Rivendare
> >   image
> 
> made the range 0 here

Sweet. Thinking about it more we don't really need a dummy caster here so I'll follow this up: https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2553

**Review by YakaryBovine — APPROVED** [review]
Looking good.

**Pull request merged**